library/src/main/java/com/squareup/timessquare/DayViewAdapter.java

```diff
@@ -0,0 +1,10 @@
+package com.squareup.timessquare;
+
+/**
+ * Created by Michal Klimczak on 30/11/15.
+ */
+public interface DayViewAdapter {
+
+    void makeCellView(CalendarCellView parent);
```

Comment:
two-character indent.  remove the two blank lines.  remove the current class comment and add one explaining what this is used for.